Resources/installer/sql/demo_clean_data.sql

```diff
@@ -265,36 +265,36 @@ UNLOCK TABLES;
 
 LOCK TABLES `ezcontentclass` WRITE;
 /*!40000 ALTER TABLE `ezcontentclass` DISABLE KEYS */;
-INSERT INTO `ezcontentclass` VALUES (1,'<short_name|name>',1024392098,14,1,'folder',2,1,3,1082454875,14,'a3d405b81be900468eb153d774f4f0d2',NULL,'a:2:{s:6:\"eng-GB\";s:6:\"Folder\";s:16:\"always-available\";s:6:\"eng-GB\";}',1,1,NULL,0);
-INSERT INTO `ezcontentclass` VALUES (1,'<name>',1024392098,14,3,'user_group',2,1,3,1048494743,14,'25b4268cdcd01921b808a0d854b877ef',NULL,'a:2:{s:6:\"eng-GB\";s:10:\"User group\";s:16:\"always-available\";s:6:\"eng-GB\";}',1,1,NULL,0);
-INSERT INTO `ezcontentclass` VALUES (1,'<first_name> <last_name>',1024392098,14,4,'user',2,0,3,1082018364,14,'40faa822edc579b02c25f6bb7beec3ad',NULL,'a:2:{s:6:\"eng-GB\";s:4:\"User\";s:16:\"always-available\";s:6:\"eng-GB\";}',1,1,NULL,0);
-INSERT INTO `ezcontentclass` VALUES (0,'<subject>',1052385685,14,13,'comment',2,0,3,1082455144,14,'000c14f4f475e9f2955dedab72799941',NULL,'a:2:{s:6:\"eng-GB\";s:7:\"Comment\";s:16:\"always-available\";s:6:\"eng-GB\";}',1,1,NULL,0);
+INSERT INTO `ezcontentclass` VALUES (1,'<short_name|name>',1024392098,14,1,'folder',2,1,3,1440409544,14,'a3d405b81be900468eb153d774f4f0d2','a:2:{s:6:\"eng-GB\";s:184:\"Set up a structure for storing content entities (articles, etc.). Note: The Folder content type is similar to the Article content type, but it does not allow image uploads or comments.\";s:16:\"always-available\";s:6:\"eng-GB\";}','a:2:{s:6:\"eng-GB\";s:6:\"Folder\";s:16:\"always-available\";s:6:\"eng-GB\";}',1,1,'',0);
+INSERT INTO `ezcontentclass` VALUES (1,'<name>',1024392098,14,3,'user_group',2,1,3,1440410082,14,'25b4268cdcd01921b808a0d854b877ef','a:2:{s:6:\"eng-GB\";s:49:\"Define here a structure for storing a user group.\";s:16:\"always-available\";s:6:\"eng-GB\";}','a:2:{s:6:\"eng-GB\";s:10:\"User group\";s:16:\"always-available\";s:6:\"eng-GB\";}',1,1,'',0);
+INSERT INTO `ezcontentclass` VALUES (1,'<first_name> <last_name>',1024392098,14,4,'user',2,0,3,1440410061,14,'40faa822edc579b02c25f6bb7beec3ad','a:2:{s:6:\"eng-GB\";s:231:\"Defines a structure for storing user accounts. Quick Tip: This option is only displayed to users who belong to the Administrators group. We recommend that you create users through the Administration Interface as it is less trouble.\";s:16:\"always-available\";s:6:\"eng-GB\";}','a:2:{s:6:\"eng-GB\";s:4:\"User\";s:16:\"always-available\";s:6:\"eng-GB\";}',1,1,'',0);
```

Comment:
"Quick Tip: This option is only displayed to users who belong to the Administrators group. We recommend that you create users through the Administration Interface as it is less trouble."

Where does this come from?
First statement depends on permissions so should not be mentioned here, and second statement is outdated.